lib/commands/kubernetesUndeploy.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright © 2018 Atomist, Inc.
```

Comment:
2019?

Reply:
I believe I lasted edited this in 2018. This PR has been a long time coming.